Homework2/task4.py

```diff
@@ -0,0 +1,13 @@
+# Задайте список из N элементов, заполненных числами из промежутка [-N, N]. Найдите произведение элементов на указанных позициях. 
+# Позиции хранятся в файле file.txt в одной строке одно число.
+
+n = int(input('Введите число: '))
+a = []
+p = 1
+for i in range((-n), n+1):
```

Comment:
```suggestion
for i in range(-n, n+1):
```
Так лучше, то тогда у вас получается список размерностью 2n, а по заданию должен быть только n